parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -202,6 +202,8 @@ function ensureFilepath() {
     fi
 }
 
+function retrycmd_if_failure() { retries=$1; wait=$2; shift && shift; for i in $(seq 1 $retries); do ${@}; [ $? -eq 0  ] && break || sleep $wait; done; echo Executed \"$@\" $i times; }
```

Comment:
FYI it must be this line that is breaking things, should we revert to the bash-compatible version and retry?